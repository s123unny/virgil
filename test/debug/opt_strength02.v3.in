b ../core/opt_strength02.v3 4
run
bt
info l
c
q
