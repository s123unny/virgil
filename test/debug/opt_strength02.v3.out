breakpoint|../core/opt_strength02.v3 4|0
stop|stopOnBreakpoint
bt|opt_strength02.main|../core/opt_strength02.v3|4
variable|0|this|opt_strength02|opt_strength02|false
variable|1|arg|0|int|false
stop|end
result|0
